include/oneapi/dpl/pstl/hetero/dpcpp/unseq_backend_sycl.h

```diff
@@ -183,6 +183,20 @@ struct __init_processing
 //------------------------------------------------------------------------
 // transform_reduce
 //------------------------------------------------------------------------
+template <typename... _Ts>
+std::tuple<_Ts...>
+select_from_group(sycl::sub_group G, std::tuple<_Ts...> value, unsigned int idx)
```

Comment:
Maybe have the `sub_group` be called `sg` like you have below

Reply:
Yeah done, good point